ignite/pkg/xgit/xgit.go

```diff
@@ -144,3 +143,14 @@ func Clone(ctx context.Context, urlRef, dir string) error {
 		Hash: *h,
 	})
 }
+
+// IsRepository checks if a path contains a Git repository.
+func IsRepository(path string) (bool, error) {
```

Comment:
This function was added with the intention to check if the app directory where the code is being scaffolded is a Git repository, but I'm not sure if asking the user to use an existing repository makes sense or how common that case would be to actually handling it in the CLI.

Currently scaffolding inside a Git repository will end with the CLI using it for the "initial" scaffolded code commit.